Makefile

```diff
@@ -15,17 +15,13 @@ REPOSITORY_NAME:=$(notdir $(REPOSITORY_PACKAGE))
 BIN_DIRECTORY := ${ROOT_DIRECTORY}/_bin
 PATH := ${PATH}:${BIN_DIRECTORY}
 
-ifdef TRAVIS_TAG
-	VERSION_BASE:=$(TRAVIS_TAG)
-else
-	VERSION_BASE:=$(shell git describe --abbrev=0 --tags 2> /dev/null || echo 'dblp.0.0.0')
-endif
-VERSION_BASE:=$(VERSION_BASE:dblp.%=%)
+
```

Comment:
Maybe worth to add a comment at the beginning of the makefile to explain VERSION is expected to be set in the env.